app/controllers/token_auth/api/payloads_controller.rb

```diff
@@ -36,6 +36,7 @@ def create
         payload = Payload.new(entity_id: entity_id)
         payload.save params[:data]
 
+        headers["Errors"] = payload.errors.join(", ")
```

Comment:
I'm compromising here: rather than reject the whole payload, accept the valid resources and reject the rest.